### DISCUSSION TITLE
Set "title" HTML attribute for a value in table column

### DISCUSSION THREAD

**Discussion by amirzpr** [filamentphp/filament#435] [Help]
Hi.
I want to set `title` HTML attribute for values of a table column so when I hover the pointer over them, the desired text shows up.
An example of what I want to achieve is displayed in the image below.
How can I do that in filament? Is it possible?

![image](https://user-images.githubusercontent.com/45946337/120099679-0cbb1b80-c152-11eb-8b74-49439e9686d7.png)

**Comment by dechoiliev** [comment]
Hi, I created it with tailwindcss, alpinejs and custom view from this example [https://codepen.io/t7team/pen/XWdyVyB](url)

**Comment by danharrin** [comment]
I'd accept a PR for `->title()` as a native customization method.
